### PR TITLE
m1m3 tables

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

**Review by tribeiro — APPROVED** [review]
Looks good! Please, rebase before merging!

File: python/lsst/ts/xml/tables/m1m3/__init__.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# This file is part of ts_xml
```
Comment (wvreeven) [inline python/lsst/ts/xml/tables/m1m3/__init__.py:1]: Please terminate the sentence in a `.`.

File: python/lsst/ts/xml/tables/m1m3/__init__.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#
+# Developed for the LSST Data Management System.
```
Comment (wvreeven) [inline python/lsst/ts/xml/tables/m1m3/__init__.py:3]: We have been using the Vera C. Rubin Observatory version for over two years now. Please update this line.

File: python/lsst/ts/xml/tables/m1m3/fa_table.py [hunk 3]
```diff
@@ -0,0 +1,2791 @@
+# This file is part of M1M3 GUI.
+#
+# Developed for the LSST Telescope and Site Systems.  This product includes
```
Comment (wvreeven) [inline python/lsst/ts/xml/tables/m1m3/fa_table.py:3]: We have been using the Vera C. Rubin Observatory version for over two years now. Please update this line.

File: python/lsst/ts/xml/tables/m1m3/fa_table.py [hunk 4]
```diff
@@ -0,0 +1,2791 @@
+# This file is part of M1M3 GUI.
```
Comment (wvreeven) [inline python/lsst/ts/xml/tables/m1m3/fa_table.py:1]: I think this skould read `part of ts_xml.` instead.

File: python/lsst/ts/xml/tables/m1m3/fcu_table.py [hunk 5]
```diff
@@ -0,0 +1,136 @@
+# This file is part of M1M3 GUI.
+#
+# Developed for the LSST Telescope and Site Systems.  This product includes
```
Comment (wvreeven) [inline python/lsst/ts/xml/tables/m1m3/fcu_table.py:3]: We have been using the Vera C. Rubin Observatory version for over two years now. Please update this line.

File: python/lsst/ts/xml/tables/m1m3/fcu_table.py [hunk 6]
```diff
@@ -0,0 +1,136 @@
+# This file is part of M1M3 GUI.
```
Comment (wvreeven) [inline python/lsst/ts/xml/tables/m1m3/fcu_table.py:1]: I think this should read `part of ts_xml.` instead.

File: tests/test_tables_m1m3.py [hunk 7]
```diff
@@ -0,0 +1,55 @@
+# This file is part of ts_criopy.
```
Comment (wvreeven) [inline tests/test_tables_m1m3.py:1]: I think this should read `part of ts_xml.` instead.

**Review by wvreeven — APPROVED** [review]
Looks good but please see my comments.

File: tests/test_tables_m1m3.py [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+#
+# Developed for the Rubin Observatory Telescope and Site System.
```
Comment (couger01) [inline tests/test_tables_m1m3.py:3]: I think this needs Vera C. prepended to Rubin.

**Review by couger01 — APPROVED** [review]
LGTM, added one additional change on top of Wouter's requests.

**Pull request merged**